src/tree/registries/azure/SubscriptionTreeItem.ts

```diff
@@ -32,13 +34,10 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase implements IR
             this._nextLink = undefined;
         }
 
-        const armContainerRegistry = await import('@azure/arm-containerregistry');
```

Comment:
Because this class extends one from the `@microsoft/vscode-azext-azureutils` package, it cannot lazy-load it (as far as I know). As a result, we have to make the rest of the Docker extension lazy-load `SubscriptionTreeItem`.